docs/examples/table/check-strictly.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <el-radio-group v-model="treeProps.checkStrictly">
+    <el-radio-button :label="true" />
+    <el-radio-button :label="false" />
```

Comment:
It is best to use the new API writing method.
```suggestion
    <el-radio-button label="false" :value="false" />
```